src/styles/partials/_positioning.scss

```diff
@@ -0,0 +1,13 @@
+h1, h2, h3 {
```

Comment:
Why don't stick to what browser provides, i guess more often than not we are going to override them in each case